Blog post draft of 10.5 release (#26)
jellyfin/jellyfin-blog: content/posts/jellyfin-10.5.md

+++
title = "Jellyfin 10.5"
subtitle = "A step closer to the best experience with your media"
author = "Fernando"
githubusername = "ferferga"
date = "2020-03-06"
justify = "center"
+++

What would you think if I tell you that 10.5 is probably the biggest release we've done? Fasten your seat belts and get ready because the numbers are big:
**Over 200 contributions from the community were merged and over 500 issues were fixed** since 10.4.3 (the latest stable version), both in Jellyfin and Jellyfin Web,
the interface you access using your browser.

> Although we tried hard, we are even unsure if we got covered everything that's new. We hope so. Anyway, you're advised to take some popcorn before continuing. Let's dive in!

Reviewer: The english here seems a bit off. I suggest instead using " we are unsure if we covered everything that is new." It has to words that break the flow right now.

"We hope so." should be changed to "We sure hope we have!" and the final sentence here should be changed to "Grab some popcorn and get ready to dive in!"

> We also want to apologise for the late release: We wanted to make 10.5 possible this Christmas, but we always found that something wasn't complete or finished yet.

Reviewer: The use of the `:` is not only incorrect, (it should be a `;`) we should just replace it with a period. Both `:` and `;` are not proper form with this informal style of writing you are aiming for.

> We hope you will forgive us (send everyone who contributed a cake if you do, please!) after trying it!

Reviewer: As I pointed out in another comment, this informal style should not contain parentheses. But since its meant to be a joke and there is no good way to do it in a separate sentence, in this case I'd suggest "We hope you will forgive us after trying it (and maybe send us some cakes)!"

# Key Features

## Let your content shine like never before

> If you're like us, you probably want to have your content showcased as beautiful as possible, right? At least, everyone who contributed to the web interface were really comitted into this, so I it seems that, at least, they want to:

Reviewer: "so I it seems that" is broken english. We should fix this grammar mistake.

Author: Thanks for noticing the typo and for the wording proposals!

#### Jellyfin 10.4.3

<img src="/images/10.5-release/old.png" name="Old item details page" />

#### Jellyfin 10.5

<img src="/images/10.5-release/new.png" name="New item details page" />

*With 10.5, we wanted to reimagine Jellyfin's Web interface*

> Posters are now as relevant as the description of the element and backgrounds introduce you to your content. In fact, I thought that I was the jungle when

Reviewer: "In fact, I thought that I was in the jungle when..." We are missing the in now making the sentance improper.

Author: Thanks again for your eyesight in this! :)

> I took that screenshot of the *Jungle's Book* wallpaper. Do you feel it like me?

Reviewer: "Do you feel it like me?" Seems off. I'd suggest "Do you also feel it?"

> Notice also that the header of the UI has been reworked a little bit in a *slightly* lighter color (instead of the plain color used by the theme)

Reviewer: "Notice also" feels wrong. I suggest "Also, notice".

Additionally, Id suggest changing the wording of the rework portion of this sentence to "the UI has bee reworked to have a slightly lighter header" and just remove the text in the parentheses. 

For stuff like this, if its not important enough to not be in parentheses it should not be present at all. It's bad form.

> Shoutout to all the Web team for their unvaluable contributions in mking this happen, specially to [Julien](https://github.com/mrtimscampi) and 

Reviewer 2: Shouldn't this be invaluable? (I'm not a native speaker, so please correct me if this is wrong)

Reviewer 3: "invaluable" is correct here.

Author: Oof, thanks!

> Shoutout to all the Web team for their unvaluable contributions in mking this happen, specially to [Julien](https://github.com/mrtimscampi) and 

Reviewer: "Unvaluable" means "of no value". You want "invaluable" which means "of value so great it cannot be determined".

English is weird yo.

[grafixeyehero](https://github.com/grafixeyehero).

You can check the redesign in our [demo server](https://demo.jellyfin.org/stable)

## Faster than the sound and cleaner than a surgery room

> As you already know, we're still a really young community that want to provide the best self-hosted media server out there. As with every release,

Reviewer: It's not good to make assumptions or bold statements in informal english like this. I'd suggest "As we hope you know, we are still a relatively young community and our goal is to provide"

Author: ``might`` just popped in my mind right now, I think it fits better.

> we tried to clean up and optimize the old code that we inherited when we forked Emby's code. And that's what we have been doing since

Reviewer: Don't say the Emby code is old. We dont want the negative connotation there even if we all feel this way. It just ends up being oddly negative in a post that should not have any in it.

It's not good to start a sentence with "And" in anything official, formal or informal. I'd instead suggest we start the sentence with "That's exactly what we have been doing" instead.

Reviewer 4: I would avoid mentioning Emby in official communications at all, since we want to shed the image of "Emby fork".

> the day we released 10.4.3. A lot of this work will pass unnoticed to many of you, but they're not less important. I'll get to that in a moment.

Reviewer: We know we released on a day. Drop it in informal situations! :)

Id also suggest the second sentence be changed to something almost entirely different. "Most of this work will go unnoticed by you, but they are still important for the project to keep being so successful!"

No cock teasing. It's inconsiderate, especially since you get right into what you were previously describing anyways. Just drop the "I'll get to that in a moment" outright.

Let's start with the frontend, as it's what you see everyday in the end. A lot of work has been done to simplify the code and ease the transition

> to more modern frameworks, which will be really useful in the future for making more awesome things with the interface of the clients. For you, end users, this probably is all rubbish because there is nothing you feel that affects you right now. Plain wrong. Just see this video:

Reviewer 2: I would rephrase this. Maybe something along the lines: While the benefits are not directly visible in a comparision of screenshots, they already paid off in increased application speed.

> to more modern frameworks, which will be really useful in the future for making more awesome things with the interface of the clients. For you, end users, this probably is all rubbish because there is nothing you feel that affects you right now. Plain wrong. Just see this video:

Author: Fixed already

<video autoplay muted loop preload="none" id="preview">
    <source src="/images/10.5-release/speed.mp4" type="video/mp4">
</video>

Not only you're seeing the speed differences, you're seeing the new animations! Images will fade in every time you move to another page. You can disable that in settings if you want to come back the old behaviour though.

The old-standing issue that  many of you had (by the way, introduced by the fellow writer of this post) caused by the images has been fixed in this version,
although we want to keep improving in that matter, as we're still not completely happy with the current implementation. 

That's pretty much what you'll notice at first sight for web anyway. A lot of not-that-relevant improvements, translations and bug fixes have been added. Consider that they're easter eggs, so happy finding! Ah, and we also added keyboard support for TVs and improved the compatibility with old browsers (just in case)!

> Okay, server time then! This time we focused heavily in migrating to .NET Core 3.0 (the framework we use for building the server's core).

Reviewer: "heavily on" not "in". And no parentheses!

Try "This time we focused heavily on migrating to .NET Core 3.0 which is the framework the server is built on.

> Although this seems boring, let's see some benchmarks from the hex decoder (used in server's tasks):

Reviewer: Though this might seem boring, lets look at some hex decoder benchmarks (used all over the server's code):

```
|                         Method |     Mean |    Error |   StdDev |   Gen 0 | Gen 1 | Gen 2 | Allocated |
|------------------------------- |---------:|---------:|---------:|--------:|------:|------:|----------:|
|                New hex decoder | 10.81 us | 0.079 us | 0.074 us |  0.1068 |     - |     - |     336 B |
|                Old hex decoder | 23.57 us | 0.055 us | 0.051 us | 11.2305 |     - |     - |   35272 B |
```
*Shoutout to [Bond-009](https://github.com/Bond-009) for this!*

That's only a tiny example of what we have been cooking and the groundwork preparation we've been doing for upcoming versions.

> However, there are some key improvements here, which are mostly focused in performance:

Reviewer: However, the majority of improvements have been focused on performance and bug fixes as usual:

* Added support for embedded attachments in media files (#1838)
* Fixed Play To order for playlists
* Raspbery Pi should have now full hardware-acceleration capabilities when encoding and decoding (older models, not in 4 yet)
* General improvements to the transcoding speed and reliability
* Reliability improvements
* Tons of bug fixes

## Release Notes

> All this blog post focused heavily in the server and the web client, as they're the most used ones. However, we also have a bunch of apps that will receive

Reviewer: While this blog post has been focused on the server and web client, upgrading your server will also benefit other clients as well"

and then drop the keep an eye on link later on in the sentence.

this update as well, so [keep an eye](https://github.com/jellyfin) on our GitHub.

> Detailed release notes available on the [GitHub's server repository](https://github.com/jellyfin/jellyfin/releases/tag/v10.5.0).

Reviewer: are available

## We're not done yet

> 

Reviewer 5: Would you like to add a few notes here for the documentation updates? We have the new network section with updated reverse proxy sections with a copypasta traefik config, the hwa has been updated. Brand new pages include the codec page, and custom css page.

> As [Joshua explained in his blog post](https://jellyfin.org/posts/jellyfin-in-2019/), we're really excited to see how our whole community is evolving,

Reviewer: growing, not evolving. Try something like "we are really excited to see our community continue to grow"

Normally informally uses contractions, but "we're really" is offputting because its two "r" sounds back to back, hence the dropping of it.

> and we're impressed by how far we got since we first started. Things are going really quick, and this release just proves that.

Reviewer: "Things are still moving really quick and this release is as good example of that". No definitive statements in informal english!

> In this post, I mentioned some of the people that contributed a lot of making this version possible. However, it's completely unfair

Reviewer: In this post I named a select few contributors that really helped make this release possible, however that is not to downplay the work of all the contributors that were not named."

Drop the rest of the sentence with the community bit. It should be understood by now and repeating it makes it more "formal"

> if only they're mentioned, as this has been a community work. That's why I want to take advantage of this blog post to say thank you, in behalf of all the project

Reviewer: "That's why I want to take the rest of the post to say thank you on behalf of the entire project and community to every single contributor out there.

Thank you very much for everything from all our translators, financial sponsors, bug fixers, and helpers in the forums, chat, and github."

Never use "/" in anything official that's informal or formal. informal and formal English follows speaking and you cant speak a "/" without sounding like a douche lol

leaders, to every single one of you out there: thank you very much for the sponsoring, thank you very much for the code contributions, bug reporting, testing...

> Thank you very much to all our translators, reddit/matrix/forum users. I also want to say a big *Thank You* to (also) a majority of users, those who simply download this software and never need support, neither suits their needs and never came back; also thank you to you who (might) be reading this and you're hearing about us right now: the solely reason of taking some time to take a look at what we do means that you care.

Reviewer 2: > neither suits their needs and never came back

I don't get what that means

Reviewer 3: Agreed, this bit seems a bit *too* flowery for me personally. The thanks is warranted, but a "thank you to everyone in the community - not just the contributors, but everyone who helps Jellyfin by using, translating, and encouraging people to use it" sounds more reasonable/professional IMO.

Author: That's exactly what I wanted to tell but with a better wording, i couldn't find a better one at the moment. Thanks!

> Thank you very much to all our translators, reddit/matrix/forum users. I also want to say a big *Thank You* to (also) a majority of users, those who simply download this software and never need support, neither suits their needs and never came back; also thank you to you who (might) be reading this and you're hearing about us right now: the solely reason of taking some time to take a look at what we do means that you care.

Reviewer: As the first sentence is covered in the last review, the rest is covered here.

"I also want to say a big thank you to the users who have simply downloaded JF regardless of it fitting your needs. We hope that you will stick around and see what else we have in store for the future!"

Instead of the whole mass of commands, colons, and semicolons you have there now.

Author: Also fixed in the new commit, you reviewed some old parts :)

> We're so proud of our community. It's our reason to exist and why we're way beyond where we expected a while back.

Reviewer: "and why we're way beyond where we expected a while back." should be "and we are so far beyond what we expected when we started."

Author: But we're saying we're where we are **because** of the community

> Let's keep fuelling this rocket together!

Reviewer: Rocket? Where did this come from? I'd just drop the sentence outright since the sentiment has been repeated a lot by now.

Author: Any synonym suggestion? I would really like to have this kind of phrase here, I don't see the need of removing it. It recalls what we mention before about our community, I see it as a motivation boost in fact 😋.

Reviewer 2: I'm also confused by the rocket and would drop the sentence. I don't think we ever used a rocket as metapher for anything before.

Reviewer 6: If you want to keep a similar sentence here you could say something like 

> Let's keep working together to keep this train rolling!

It's a much more common saying, and flows a bit better in this context

> With all that said, I wish you a happy weekend and a happy *jellyfining*!

Reviewer: You can drop the second "a" here. Make it "I wish you a happy weekend and happy \*jellyfinning!*"

> PS: If you're still making plans for this weekend, cancel them. And if you have appointments already fixed, cancel them. No exceptions.

Reviewer 2: Do we really want `jellyfining` to become a verb? 

Also, the PS feels weird to me, maybe we should rather wish everyone a great weekend and to enjoy Jellyfin with their friends and family or something like that :D

Reviewer 3: Yea I don't like verb-ing the name, it sounds kinda corny personally. I can't recall if I used it this way, but if I did, I take it back :laughing:

Author: Done. I don't know if we want emojis there, but I added one, opinions on this please :)

> PS: If you're still making plans for this weekend, cancel them. And if you have appointments already fixed, cancel them. No exceptions.

Reviewer: I'd just drop this line. Telling users what to do isn't that funny in any situation unfortunately.

Author: Perhaps the latest one is better? You reviewed the older commit

+++
title = "Jellyfin 10.5"
subtitle = "A step closer to the best experience with your media"
author = "Fernando"
githubusername = "ferferga"
date = "2020-03-06"
justify = "center"
+++

Hello Jellyfin users out there! Welcome back to our blog!

> What would you think if I tell you that 10.5 is probably the biggest release we've done? Fasten your seat belts and get ready because the numbers are big:

Reviewer 6: What would you think if I **told** you that 10.5 is probably the biggest release we've **ever** done? Fasten your seat belts and get ready because the numbers are big:

**Over 200 contributions from the community were merged and over 500 issues were fixed** since 10.4.3 (the latest stable version), both in Jellyfin and Jellyfin Web,

> the interface you access using your browser.

Author: I will leave a mix with the suggestions of others in the following commit. I'll leave the ``the interface you access using your browser.`` sentence, because newcomers might not know what it is just at the beginning.

Although we tried hard, we are even unsure if we got covered everything that's new. We hope so. Anyway, you're advised to take some popcorn before continuing. Let's dive in!

We also want to apologise for the late release: We wanted to make 10.5 possible this Christmas, but we always found that something wasn't complete or finished yet.
We hope you will forgive us (*send everyone who contributed a cake if you do, please!*) after trying it!

# Key Features

## Let your content shine like never before

> If you're like us, you probably want to have your content showcased as beautiful as possible, right? At least, everyone who contributed to the web interface were really comitted into this, so I it seems that, at least, they want to:

Reviewer 6: Maybe something like this:

> If you're like us, you probably want your content **to be** showcased as **beautifully** as possible, right? Our volunteers working on the web interface seem to agree, and have been very committed to this goal:

Author: I think that contributors is better, basically because it's the widely known term for people who work in open source software. At least based on what I've seen in other places.

#### Jellyfin 10.4.3

<img src="/images/10.5-release/old.png" name="Old item details page" />

#### Jellyfin 10.5

> <img src="/images/10.5-release/new.png" name="New item details page" />

Author: Not sure how it will scale in the page though, content is really stretched there

*With 10.5, we wanted to reimagine Jellyfin's Web interface*

Posters are now as relevant as the description of the element and backgrounds introduce you to your content. In fact, I thought that I was the jungle when
I took that screenshot of the *Jungle's Book* wallpaper. Do you feel it like me?

Notice also that the header of the UI has been reworked a little bit in a *slightly* lighter color (instead of the plain color used by the theme)

Shoutout to all the Web team for their invaluable contributions in mking this happen, specially to *[Julien](https://github.com/mrtimscampi)* and 

> *[grafixeyehero](https://github.com/grafixeyehero)*.

Reviewer 4: ```suggestion
Jellyfin 10.5.0 is the start of a major user interface rework that will be spread over the coming releases.

This release starts with a complete overhaul of the details screen, where posters are now as relevant as the description of your item and backdrops are front and center.

The header also has a tighter layout and a lighter color in our default Dark theme, to make the page less monotone.

We'd like to thank every contributor to the web client for their invaluable contributions in making this happen, especially to *[MrTimscampi](https://github.com/mrtimscampi)* and *[grafixeyehero](https://github.com/grafixeyehero)*.
```

Rewording of a few paragraphs to be less conversational and familiar. Also use my nickname, since that's what I appear as on all the community platforms (It's easier for people to make the link, even if I don't care about using my real name).

Author: I feel that removing most of that is maybe too much, I think we need to give more selling points and an appeal to the updates, so people *want* to upgrade/switch, not *need to* (see #21)

I will make some of your proposed changes in this section in the wording of some elements, so take a look at the commit that I'm going to push.

> You can check the redesign in our [demo server](https://demo.jellyfin.org/stable)

Reviewer 4: ```suggestion
You can try out the redesign on our [demo server](https://demo.jellyfin.org/stable).
```

> ## Faster than the sound and cleaner than a surgery room

Reviewer 6: Faster than the **speed of** sound

As you already know, we're still a really young community that want to provide the best self-hosted media server out there. As with every release,
**we tried to clean up and optimize the old code** that we inherited when we forked Emby's code. And that's what we have been doing since
the day we released 10.4.3. A lot of this work will pass unnoticed to many of you, but they're not less important. I'll get to that in a moment.

Let's start with the frontend, as it's what you see everyday in the end. A lot of work has been done to simplify the code and ease the transition
to more modern frameworks, which will be really useful in the future for making more awesome things with the interface of the clients. While
all the benefits of this are not visible for you in the daily use just yet, it's a necessary step towards a better application. In fact, it already paid off in application speed. See this video:

<video autoplay muted loop preload="none" id="preview">
    <source src="/images/10.5-release/speed.mp4" type="video/mp4">
</video>

> Not only you're seeing the speed differences, you're seeing the new animations! Images **will fade in** every time you move to another page. You **can** disable that in settings if you want to come back the old behaviour though.

Reviewer 4: You cannot really disable it, currently the switch in the settings is to change the transition speed between two hardcoded ones.

Author: Thanks for pointing this, didn't know!

> The old-standing issue that many of you had (by the way, introduced by the *fellow writer* of this post 🙄) caused by the images has been fixed in this version

Reviewer 4: Which issue are you referring to ? The reader might not know, they might be a new user looking at what the latest release brings, etc.

(although we want to keep improving in that matter, as we're still not completely happy with the current implementation) 

That's pretty much what you'll notice at first sight for web anyway. A lot of not-that-relevant improvements, translations and bug fixes have been added. Consider that they're easter eggs, so happy finding! Ah, and we also added keyboard support for TVs and improved the compatibility with old browsers (just in case)!

> Okay, server time then! This time we focused heavily in migrating to .NET Core 3.0 (the framework we use for building the server's core).

Reviewer 4: Should be another header section.

Reviewer 4: It's also 3.1, not 3.0 :)

Author: That's a leftover I wrote before checking haha, thought I changed it, thanks!

Author: > Should be another header section.

I don't think so because we're still talking about speed and reliability improvements

> Although this seems boring, let's see some benchmarks from the hex decoder (used in server's tasks):

Reviewer 4: I don't think the move to .NET Core 3.1 is directly related to the speed improvement shown here.

That's due to a refactoring of the hex decoder, I believe.

Author: I know, but I don't see the need in diving more into this for non-techy users, that's why I explain in a non-concise way what the hex decoder do

|                New hex decoder | 10.81 us | 0.079 us | 0.074 us |  0.1068 |     - |     - |     336 B |
|                Old hex decoder | 23.57 us | 0.055 us | 0.051 us | 11.2305 |     - |     - |   35272 B |
```
*Shoutout to [Bond-009](https://github.com/Bond-009) for this!*

That's only a tiny example of what we have been cooking and the groundwork preparation we've been doing for upcoming versions.
However, there are some key improvements here, which are mostly focused in performance:

* Added support for embedded attachments in media files (#1838)
* Fixed Play To order for playlists
* Raspberry Pi should have now full hardware-acceleration capabilities when encoding and decoding (older models, not in 4 yet)
* General improvements to the transcoding speed and reliability
* Reliability improvements
* Tons of bug fixes

## Extra, Extra!

Along the way, we also updated a lot of our documentation. Specifically, the experts have been working hard in revamping the **network/reverse proxy**
and the **hardware acceleration** configuration guides. Not only improvements have been made, but also new sections have been added to the documentation, such

> as the extensive **codec** and the **custom CSS branding** guides.

Reviewer 6: We should link directly to these specific pages in the documentation, where possible

[Check it out!](https://jellyfin.org/docs/). With this new documentation you will probably end up with a setup we couldn't even match!

## Release Notes

All this blog post focused heavily in the server and the web client, as they're the most used ones. However, we also have a bunch of apps that will receive
this update as well, so [keep an eye](https://github.com/jellyfin) on our GitHub.

Detailed release notes available on the [GitHub's server repository](https://github.com/jellyfin/jellyfin/releases/tag/v10.5.0).

## We're not done yet

As [Joshua explained in his blog post](https://jellyfin.org/posts/jellyfin-in-2019/), we're really excited to see how our whole community is evolving,

> and we're impressed by how far we got since we first started. Things are going really quick, and this release just proves that.

Reviewer 6: we're impressed by how far **we've gotten**

In this post, I mentioned some of the people that contributed a lot of making this version possible. However, it's completely unfair

> if only they're mentioned, as this has been a community work. That's why I want to take advantage of this blog post to say thank you, in behalf of all the project

Reviewer 4: It's probably be nicer to have something like "Here's the list of all the contributors to this cycle, who we'd like to thank for helping Jellyfin get better" or something.

Then add the list Josh shared the other day in alphabetical order to mention everyone by name.

Author: Would be really long I think. In this part we're also thanking people who might not have directly contributed as well.

+++
title = "Jellyfin 10.5"
subtitle = "A step closer to the best experience with your media"
author = "Fernando"
githubusername = "ferferga"
date = "2020-03-06"
justify = "center"
+++

Our biggest release just yet

<!--more-->

Hello Jellyfin users out there! Welcome back to our blog!

Fasten your seat belts and get ready because the numbers are big: Since 10.4.3, **over 200 contributions** from the community were merged
and **over 500 issues** were fixed**Over 200 contributions from the community were merged and over 500 issues were fixed** both in Jellyfin and Jellyfin Web, the interface you access using your browser.

We also want to apologise for the late release: We wanted to make 10.5 possible this Christmas, but we always found that something wasn't complete or finished yet.
We hope you will forgive us after trying it (*and maybe send us some cakes*)!

> Although we tried hard, we are even unsure if we got covered everything that's new. We sure hope we have!. Anyway, you're advised to take some popcorn before continuing. Grab some popcorn and get ready to dive in!

Reviewer 6: Here's a rephrasing that I tried to keep close to the tone of the original but I think flows a little bit better:

> Although we tried hard, there's so much packed into this release that we are still unsure if this blog post actually manages to cover all the awesome new features. We sure hope it does! Either way, there's a lot of content to cover so grab some popcorn and get ready to dive in!

Author: I really like this one, thank you!

# Key Features

## Let your content shine like never before

> If you're like us, you probably want to have your content showcased as beautiful as possible, right? At least, everyone who contributed to the web interface were really comitted to this so it seems they wanted to:

Reviewer 6: * beautiful -> beautifully
* everyone who contributed to the web interface -> our web interface contributors

#### Jellyfin 10.4.3

<img src="/images/10.5-release/old.png" name="Old item details page" />

#### Jellyfin 10.5

<img src="/images/10.5-release/new.png" name="New item details page" />

*With 10.5, we wanted to reimagine Jellyfin's Web interface*

Posters are now as relevant as the description of the element and backgrounds introduce you to your content. In fact, I thought that I was in the jungle when

> I took that screenshot of the *Jungle's Book* wallpaper. Do you also feel it?

Reviewer 6: Jungle's Book -> Jungle Book's

Author: I'm using the original name of the movie for the next commit anyway, with quotes, so it's understandable for everyone. I tried to do a literal translation from Spanish but I think it's better to keep it like the original

Also notice that the header of the UI has been reworked to have a *slightly* lighter header

> Shoutout to all the Web team for their invaluable contributions in mking this happen, specially to *[MrTimscampi](https://github.com/mrtimscampi)* and 

Reviewer 6: * all the Web team -> everyone on the Web team
* mking -> making
* specially -> especially

*[grafixeyehero](https://github.com/grafixeyehero)*.

You can try out the redesign on our [demo server](https://demo.jellyfin.org/stable)

## Faster than the speed of sound and cleaner than a surgery room

> As you might know, we are still a relatively young community and our goal is to provide the best self-hosted media server out there. Like usual, we did what we can to clean up the code and a lot of this work will pass unnoticed to many of you, but they're not less important. I'll get to that in a moment.

Reviewer 6: * not less important
* not any less important

Let's start with the web client, since it's the most obvious to you. A lot of work has been done to simplify the code and ease the transition

> to more modern frameworks, which will be really useful in the future for making more awesome things with the interface of the clients. While

Reviewer 6: with the interface of the clients -> for our client interfaces

> all the benefits of this are not visible for you in the daily use just yet, it's a necessary step towards a better application. In fact, it already paid off in application speed. See this video:

Reviewer 6: are not visible for you in the daily use just yet -> will not be visible to you in your daily use just yet

<video autoplay muted loop preload="none" id="preview">
    <source src="/images/10.5-release/speed.mp4" type="video/mp4">
</video>

> Not only it is faster, we have new animations too! Images will fade in every time you change pages. If you don't like the new animations they can be modified in settings

Reviewer 6: it is faster -> is it faster

Author: Why? There is no question here that needs an switch between subject and verb. Explain in detail please, maybe I'm missing something 😅.

Reviewer 6: Honestly I don't know the explicit grammar rules well enough to explain why it's wrong, I just know it as a native English speaker ahaha

> We finally fixed the long standing issue many of you had with images (which I introduced, didn't mean to cause it!). Although the issue is no longer around us, we still aren't happy with the end result and hope to further improve it going forward.

Reviewer 6: * long standing -> longstanding (one word)
* no longer around us -> no longer around (remove us)

The long-standing issue that many of you had ( of this post 🙄) caused by the images has been fixed in this version

> (although we want to keep improving in that matter, as we're still not completely happy with the current implementation) 

Reviewer 6: This sentence is duplicated.

Author: True, forgot to remove the old one, thanks for pointing that out 😅.

That's pretty much all for the web interface this release. The rest of the work has primarily been translations and bug fixes. Have fun finding out which ones we solved! Oh, and before I move on to the server, we added better keyboard support for TVs and improved compatibility with old browsers.

Okay, server time then! This time we focused heavily on migrating to .NET Core 3.1 which is the framework the server is built on.

> Although this might be not interesting at all, let's see some benchmarks from the hex decoder (used in server's tasks):

Reviewer 6: interesting at all -> interesting to everyone
